### ISSUE TITLE
HTML登陆界面点击按钮无反应

### ISSUE THREAD

**Issue by RandomFkName** [RandomFkName/-#1]
我利用AJAX提交信息，在页面上拼接输入的字符串。登录按钮点击后页面没有反应，请大哥们帮我看看代码！万分感谢！！！！！
[login - 副本.txt](https://github.com/user-attachments/files/16380203/login.-.txt)php
[login-ajax - 副本.txt](https://github.com/user-attachments/files/16380206/login-ajax.-.txt)html

<!DOCTYPE html>
<html lang="zh">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script type="text/javascript" src="jquery-3.7.1.min.js"></script>
    <title>登录页面</title>
    <style>
        body {
            font-family: Arial, sans-serif;
            background-color: #f4f4f4;
            margin: 0;
            padding: 0;
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100vh;
        }
        .login-container {
            background-color: white;
            padding: 20px;
            border-radius: 5px;
            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
        }
        .login-container h2 {
            text-align: center;
        }
        .form-group {
            margin-bottom: 15px;
        }
        .form-group label {
            display: block;
            margin-bottom: 5px;
        }
        .form-group input {
            width: 100%;
            padding: 10px;
            border: 1px solid #ddd;
            border-radius: 4px;
        }
        .form-group input[type="text"]:focus {
            outline: none;
            border-color: #5cb85c;
        }
        .form-group input[type="password"]:focus {
            outline: none;
            border-color: #5cb85c;
        }

        button {
            background-color: #5cb85c;
            color: white;
            border: none;
            cursor: pointer;
        }
        button:hover {
            background-color: #4cae4c;
        }
    </style>

    <script>
        function doPost() {
            var username = $('#username').val();
            var password = $('#password').val();
            var vcode = $('#vcode').val();
            var param = "username=" + username + "&password=" + password + "&vcode=" + vcode;
            alert(param);
        }
    </script>
</head>
<body>
    <div class="login-container">
        <h2>用户登录</h2>
            <div class="form-group">
                <label for="username">用户名:</label>
                <input type="text" id="username" name="username" required>
            </div>
            <div class="form-group">
                <label for="password">密码:</label>
                <input type="password" id="password" name="password" required>
            </div>
            <div class="form-group">
                <label for="vcode">验证码：</label>
                <input type="text" id="vcode" name="vcode" required>
            </div>
            <div class="form-group">
                <button type ="submit" onclick="doPost()">登录</button>
                <a href="register.php">注册</a>
            </div>
    </div>
</body>
</html>
